<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="p_lookbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>400</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>상품 조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iv_lookbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>210</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>재고 조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="product_addbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>355</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>상품 추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="product_changebtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>310</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>상품 수정</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pcancelbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>300</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>지정취소</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>22</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="acancelbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>350</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>전체취소</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paybtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>401</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>24</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>결제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="product_deletebtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>265</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>상품 삭제</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>471</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="salebtn">
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>판매</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lookbtn">
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitbtn">
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>종료</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="salelist_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>4</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>판매목록</string>
    </property>
   </widget>
   <widget class="QLabel" name="basketlist_label">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>4</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>장바구니</string>
    </property>
   </widget>
   <widget class="QLabel" name="cartegpry_label">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>카테고리</string>
    </property>
   </widget>
   <widget class="QLabel" name="payhistory_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>결제내역</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="product_table">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>50</y>
      <width>319</width>
      <height>339</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>600</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>55</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Qt
POS Program</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="basketlist_table">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>50</y>
      <width>242</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="inventory_addbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>165</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>발주</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="iv_table">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>59</y>
      <width>461</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="pay_table">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>39</y>
      <width>461</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="soon_t_table">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>119</y>
      <width>461</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="trashbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>10</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>폐기 조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="soon_trashbtn">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>유통기한 임박 상품</string>
    </property>
   </widget>
   <widget class="QLabel" name="today_date_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>40</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>ㅎㅇㅋㅋ</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="t_table">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>119</y>
      <width>461</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="now_trashbtn">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>유통기한 지난 상품</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="now_t_table">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>119</y>
      <width>461</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="t_btn">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>폐기 상품</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="p_table">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>59</y>
      <width>461</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="searchbox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>22</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="p_searchbtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>22</y>
      <width>66</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchedit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>22</y>
      <width>179</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="iv_searchbtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>22</y>
      <width>66</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QPushButton" name="journalbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>110</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>저널 조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graphbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>65</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>매출그래프</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graph_weekbtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>110</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>주 매출 통계</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graph_monthbtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>110</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>월 매출 통계</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graph_yearbtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>110</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>연 매출 통계</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>431</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="journal_date_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>조회일자</string>
    </property>
   </widget>
   <widget class="QPushButton" name="journal_cal_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>달력</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="journal_date_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="journal_lookbtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>24</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>조회</string>
    </property>
   </widget>
   <widget class="QTableView" name="background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>471</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="journalLeftbar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>308</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="journalRightbar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>161</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="journalTopbar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>471</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="journalTopbar_label1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>조회날짜</string>
    </property>
   </widget>
   <widget class="QLabel" name="journalTopbar_label2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>yyyy-mm-dd</string>
    </property>
   </widget>
   <widget class="QLabel" name="journalTopbar_label3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>영업일자</string>
    </property>
   </widget>
   <widget class="QLabel" name="journalTopbar_label4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>yyyy-mm-dd</string>
    </property>
   </widget>
   <widget class="QLabel" name="journalTopbar_label5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>거래NO</string>
    </property>
   </widget>
   <widget class="QLabel" name="journalTopbar_label6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0001</string>
    </property>
   </widget>
   <widget class="QLabel" name="journalTopbar_label7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>합계금액</string>
    </property>
   </widget>
   <widget class="QLabel" name="journalTopbar_label8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2000</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>p_lookbtn</zorder>
   <zorder>iv_lookbtn</zorder>
   <zorder>product_addbtn</zorder>
   <zorder>product_changebtn</zorder>
   <zorder>pcancelbtn</zorder>
   <zorder>combobox</zorder>
   <zorder>acancelbtn</zorder>
   <zorder>paybtn</zorder>
   <zorder>product_deletebtn</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>salelist_label</zorder>
   <zorder>basketlist_label</zorder>
   <zorder>cartegpry_label</zorder>
   <zorder>payhistory_label</zorder>
   <zorder>product_table</zorder>
   <zorder>logo</zorder>
   <zorder>basketlist_table</zorder>
   <zorder>inventory_addbtn</zorder>
   <zorder>iv_table</zorder>
   <zorder>pay_table</zorder>
   <zorder>soon_t_table</zorder>
   <zorder>trashbtn</zorder>
   <zorder>soon_trashbtn</zorder>
   <zorder>today_date_label</zorder>
   <zorder>t_table</zorder>
   <zorder>now_trashbtn</zorder>
   <zorder>now_t_table</zorder>
   <zorder>t_btn</zorder>
   <zorder>p_table</zorder>
   <zorder>searchbox</zorder>
   <zorder>p_searchbtn</zorder>
   <zorder>searchedit</zorder>
   <zorder>iv_searchbtn</zorder>
   <zorder>journalbtn</zorder>
   <zorder>graphbtn</zorder>
   <zorder>graph_weekbtn</zorder>
   <zorder>graph_monthbtn</zorder>
   <zorder>graph_yearbtn</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>journal_date_box</zorder>
   <zorder>journal_cal_btn</zorder>
   <zorder>journal_date_edit</zorder>
   <zorder>journal_lookbtn</zorder>
   <zorder>journalLeftbar</zorder>
   <zorder>journalRightbar</zorder>
   <zorder>journalTopbar</zorder>
   <zorder>journalTopbar_label1</zorder>
   <zorder>journalTopbar_label2</zorder>
   <zorder>journalTopbar_label3</zorder>
   <zorder>journalTopbar_label4</zorder>
   <zorder>journalTopbar_label5</zorder>
   <zorder>journalTopbar_label6</zorder>
   <zorder>journalTopbar_label7</zorder>
   <zorder>journalTopbar_label8</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitbtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>722</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
